workflows/src/test/kotlin/com/r3/developers/csdetemplate/flowexample/workflows/MyFirstFlowTest.kt

```diff
@@ -1,44 +1,48 @@
 package com.r3.developers.csdetemplate.flowexample.workflows
 
-import com.r3.developers.csdetemplate.flowexample.workflows.MyFirstFlow
-import com.r3.developers.csdetemplate.flowexample.workflows.MyFirstFlowResponder
-import com.r3.developers.csdetemplate.flowexample.workflows.MyFirstFlowStartArgs
-import net.corda.simulator.RequestData
-import net.corda.simulator.Simulator
-import net.corda.v5.base.types.MemberX500Name
-import org.junit.jupiter.api.Test
-
-class MyFirstFlowTest {
-
-    // Names picked to match the corda network in config/dev-net.json
-    private val aliceX500 = MemberX500Name.parse("CN=Alice, OU=Test Dept, O=R3, L=London, C=GB")
-    private val bobX500 = MemberX500Name.parse("CN=Bob, OU=Test Dept, O=R3, L=London, C=GB")
-
-    @Test
-    fun `test that MyFirstFLow returns correct message`() {
-
-        // Instantiate an instance of the Simulator
-        val simulator = Simulator()
-
-        // Create Alice and Bob's virtual nodes, including the Class's of the flows which will be registered on each node.
-        // We don't assign Bob's virtual node to a val because we don't need it for this particular test.
-        val aliceVN = simulator.createVirtualNode(aliceX500, MyFirstFlow::class.java)
-        simulator.createVirtualNode(bobX500, MyFirstFlowResponder::class.java)
-
-        // Create an instance of the MyFirstFlowStartArgs which contains the request arguments for starting the flow
-        val myFirstFlowStartArgs = MyFirstFlowStartArgs(bobX500)
-
-        // Create a requestData object
-        val requestData = RequestData.create(
-            "request no 1", // A unique reference for the instance of the flow request
-            MyFirstFlow::class.java, // The name of the flow class which is to be started
-            myFirstFlowStartArgs // The object which contains the start arguments of the flow
-        )
-
-        // Call the Flow on Alice's virtual node and capture the response from the flow
-        val flowResponse = aliceVN.callFlow(requestData)
-
-        // Check that the flow has returned the expected string
-        assert(flowResponse == "Hello Alice, best wishes from Bob")
-    }
-}
+//import com.r3.developers.csdetemplate.flowexample.workflows.MyFirstFlow
```

Comment:
should this file get deleted ?

Reply:
No, we will need to add it back in when Simulator UTXO work gets merged in so we can test simulator works. 
It will be removed before the beta 2 branch is cut though.